workplace/Living in Open Workspaces.md

```diff
@@ -0,0 +1,76 @@
+## Living in Open Workspaces
+
+Example: 30-45 minutes
+
+Lesson: ? min
+Guided Practice: ?-? min
+Independent Practice: ? min
+Check for Understanding: ?-? min
+
+## Prerequisites
+
+Here are topics that should be understood before this topic:
+
+- Basic terms common to workspace
```

Comment:
This seems like something that should be more fully described in another section... it is too vague to help here. This section is for listing any outlines that should have been read before this one.

Reply:
@alodahl So, I will remove the prerequisite section as there is nothing to have to be known before.